### ISSUE TITLE
Debian package of crowdsec-firewall-bouncer v0.0.15 missing

### ISSUE THREAD

**Issue by saz** [crowdsecurity/cs-firewall-bouncer#78]
I'm trying to install crowdsec-firewall-bouncer, but it seems that version 0.0.15 is missing:

```
root@machine:~# apt install crowdsec-firewall-bouncer                                                     
Reading package lists... Done
Building dependency tree                 
Reading state information... Done                                                                                                                                                                          Some packages could not be installed. This may mean that you have                                                                                                                                          
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created                                                                                                                                         
or been moved out of Incoming.
The following information may help to resolve the situation:
                                                                                                     
The following packages have unmet dependencies:                                                      
 crowdsec-firewall-bouncer : Depends: crowdsec-firewall-bouncer-nftables (= 0.0.13) but 0.0.15 is to be installed or
                                      crowdsec-firewall-bouncer-iptables (= 0.0.13) but 0.0.15 is to be installed
E: Unable to correct problems, you have held broken packages.
```
Looking at [packagecloud](https://packagecloud.io/app/crowdsec/crowdsec/search?q=crowdsec-firewall-bouncer&filter=debs&dist=) I can't find a .deb package, but [.dsc files](https://packagecloud.io/app/crowdsec/crowdsec/search?q=crowdsec-firewall-bouncer&filter=debs&filter=dscs&dist=) are available.

It looks like as if the build failed. Is there any place to check the current build status?

**Comment by buixor** [comment]
Hello,

Thanks for the report !
Please see : https://discourse.crowdsec.net/t/fresh-firewall-bouncer-install-issue/270/2

tl;dr : only install the `-iptables` or `-nftables` suffix. The package without suffix is not needed anymore and we're going to remove them from the repositories to avoid any confusion.

Please re-open if the issue persists !
